src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp

```diff
@@ -194,46 +198,83 @@ Json::Value AppKeyBindingsSerialization::ToJson(const winrt::TerminalApp::AppKey
 winrt::TerminalApp::AppKeyBindings AppKeyBindingsSerialization::FromJson(const Json::Value& json)
 {
     winrt::TerminalApp::AppKeyBindings newBindings{};
+    LayerJson(newBindings, json);

```

Comment:
I guess it theoretically could - I had all the keybinding serialization outside of AppKeyBindings, because I didn't know you could use the whole `make_self<implementation::AppKeyBindings>` thing to get at the implementation type of the AKB. So this entire file really exists to do it outside the class. I suppose I could leave the file as is, and add the methods to AKB directly. Probably more sensible that way.